### PR TITLE
PLAT-141480: Scroller - scrollbar is not very visible

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
ScrollerThumb is not very visible on Carbom, Cobalt, Copper, Electro, Titanium skins

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed color of ScrollerThumb to a more visible one

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-141480

### Comments

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

File: styles/colors-cobalt-day.less [hunk 1]
```diff
@@ -271,4 +271,4 @@
 
 // Scroller
 // ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
+@agate-scroller-scrollbar-thumb-bg-color: @agate-gray;
```
Comment (stanca-pop-lgp) [inline styles/colors-cobalt-day.less:274]: @agate-gray is still hardly visible. Maybe use @agate-accent here as well?

File: styles/colors-cobalt.less [hunk 2]
```diff
@@ -272,7 +272,3 @@
 @agate-togglebutton-bg-color: @agate-button-bg-color;
 @agate-togglebutton-selected-color: @agate-button-selected-color;
 @agate-togglebutton-selected-bg-color:  @agate-button-selected-bg-color;
-
-// Scroller
-// ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
```
Comment (stanca-pop-lgp) [inline styles/colors-cobalt.less:278]: And here you can use @cobalt-accent2.

File: styles/colors-copper.less [hunk 3]
```diff
@@ -270,7 +270,3 @@
 @agate-togglebutton-bg-color: @agate-button-bg-color;
 @agate-togglebutton-selected-color: @agate-button-selected-color;
 @agate-togglebutton-selected-bg-color:  @agate-button-selected-bg-color;
-
-// Scroller
-// ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
```
Comment (stanca-pop-lgp) [inline styles/colors-copper.less:276]: Maybe use the same color as the arrow icons. Currently it is @agate-darker-gray, but you can change it to the lighter text-color that you chose in #564. (@agate-gray)

File: styles/colors-carbon.less [hunk 4]
```diff
@@ -264,4 +264,4 @@
 
 // Scroller
 // ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
+@agate-scroller-scrollbar-thumb-bg-color: @agate-gray;
```
Comment (stanca-pop-lgp) [inline styles/colors-carbon.less:267]: @agate-text-color, maybe?

File: styles/colors-titanium.less [hunk 5]
```diff
@@ -261,4 +261,4 @@
 
 // Scroller
 // ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
+@agate-scroller-scrollbar-thumb-bg-color: @agate-gray;
```
Comment (stanca-pop-lgp) [inline styles/colors-titanium.less:264]: @agate-text-color, maybe?

File: styles/colors-cobalt-day.less (hunk 1 above)
Comment (adrian-cocoara-lgp) [reply styles/colors-cobalt-day.less:274]: Okay

File: styles/colors-cobalt.less (hunk 2 above)
Comment (adrian-cocoara-lgp) [reply styles/colors-cobalt.less:278]: 👍

File: styles/colors-cobalt-day.less [hunk 6]
```diff
@@ -271,4 +271,4 @@
 
 // Scroller
 // ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
+@agate-scroller-scrollbar-thumb-bg-color: @agate-accent;
```
Comment (daniel-stoian-lgp) [inline styles/colors-cobalt-day.less:274]: Try to find a common variable for all skins. How about @agate-text-color on all skins? If it looks alright, then you can use it in colors-base and remove @agate-scroller-scrollbar-thumb-bg-color from colors-[skin].less

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: styles/colors-cobalt-day.less (hunk 6 above)
Comment (adrian-cocoara-lgp) [reply styles/colors-cobalt-day.less]: I used `@agate-text-color` as base color, and in my opinion it looks good. For now on Copper skin it does not look so great, but after [#564 ](https://github.com/enactjs/agate/pull/564) is merged, it should be ok.

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]
💯

**Pull request merged**